Sources/GRPCCore/Metadata.swift

```diff
@@ -494,15 +494,15 @@ extension Metadata.Value: ExpressibleByArrayLiteral {
 
 extension Metadata: CustomStringConvertible {
   public var description: String {
-    String(describing: self.map({ ($0.key, $0.value) }))
+    String(describing: self.map({ ($0.key.description, $0.value.description) }))
```

Comment:
Can you add a test to verify the description is as we'd expect for a single `Metadata.Value` and for a `Metadata`?